tests/ops/functions/test_generator.py

```diff
@@ -322,7 +322,7 @@ def test_hermitian(self):
         gen = qml.generator(HermitianOp, format="hamiltonian")(0.5, wires=0)
         assert gen.name == "Hamiltonian"
 
-        expected = qml.Hamiltonian(*qml.utils.decompose_hamiltonian(HermitianOp.H))
+        expected = qml.pauli_decompose(HermitianOp.H, hide_identity=True)

```

Comment:
Why are we using `hide_identity=True`, when it was set to `False` before?

Reply:
Added 👍🏼